playpen/repo_level_awareness/agent/ast_diff/base.py

```diff
@@ -43,13 +48,13 @@ def __eq__(self, o: object) -> bool:
             return True
         return False
 
-    def __iter__(self) -> Iterator[str]:
-        return iter([v.to_dict() for _, v in self.items()])
+    # def __iter__(self) -> Iterator[str]:
```

Comment:
was this gross? I needed to convert nested objects that are a dict to a list instead as I didn't care about their keys